### ISSUE TITLE
Other social login support

### ISSUE THREAD

**Issue by Floppy** [openpolitics/votebot#83]
Allow people to log in with other social accounts. Will need to use separate committer / author for the actual changes.

**Comment by jt-nti** [comment]
This would definitely be good. At least one person has ended up not contributing to the Eastleigh manifesto due to the GitHub account requirement. (At least so far- I'm still nagging him!)

**Comment by Floppy** [comment]
I've got two I want to add in the near future; people have asked about Google and NationBuilder. What would be useful with your users do you think?

**Comment by jt-nti** [comment]
Just having an alternative would be a good start. I have so far been nagging people to participate using twitter, so they have that, but I'm not sure that's especially representative of ordinary people. Facebook seems like a reasonable candidate if that's possible but I have no idea what would be useful for younger users!

**Comment by Floppy** [comment]
Well, the big leap is from 1 to >1, after that adding more should be pretty easy. The problem is that I need to work out how to associate the git commits with someone who isn't a GitHub user. And not so much technically how, but more like what's the RIGHT way to do it to preserve the good aspects of having it all in version control?

**Comment by jt-nti** [comment]
I did wonder how you would preserve the link between users and git commits. Would setting the author work? For example, committer is the votebot account and the author is something that maps to the logged in user?

**Comment by Floppy** [comment]
Yep, that's the plan, and it's easy enough, but I'm wondering how to best identify the author. Probably a link to their profile on Groupthink, I guess.
